### ISSUE TITLE
Fix keyboard navigation in browser tree

### ISSUE THREAD

**Issue by micser** [micser/musicco#30]
- when the tree is filtered, keyboard navigation just does not work anymore ==> It does, it is just a css issue to be solved with https://github.com/mar10/fancytree/wiki/TutorialTheming
- <strike>uncheck old, search for something that triggers old and finds nothing or finds something ==> nodata is displayed</strike>
- <strike>Add a "got to artist" entry in the context menu on the search panel</strike> done
- <strike>tree filtering is super slow, maybe improve the normalising of the search query to help?</strike> The problem wasn't node matching but the event queueing! Solved by dropping filtering events if the user is still typing.
